requests-for-comments/2022-06-10-nail-exports.md

```diff
@@ -96,13 +96,14 @@ Subscriptions via slack are essentially Image exports but delivered via slack. W
 
 ---
 ### CSV / Excel:
-| insights ✅ | dashboards ✅ | person ✅ | cohorts ✅ | groups ✅ |
-|------------|--------------|----------|-----------|----------|
+| insights ✅ | dashboards ✅ | events ✅ | person ✅ | cohorts ✅ | groups ✅ |
+|------------|--------------|----------|-----------|-----------|----------|
 
-Resources can be exported directly to a CSV
+Resources can be exported directly to a CSV. This may be commonly done adhoc (e.g. from some in-memory filter configured on the frontend.
 
 #### Technical considerations
 - [ ] Exporting 10000s of rows can be costly or even impossible syncronously - it needs to be possible to do this in the background and send to the user when complete
+  - [ ] This could also incur very high costs. Perhaps we still need some upper bounds.
 - [ ] TTLs: Exported data can include very sensitive information. Ideally we should only keep the exported file for long enough to satisfy the user's request
```

Comment:
Should we include data control options around this (e.g. this requiring some scope of user access)? Exporting all raw data via UI might be potential security risk at some organizations.